src/token/erc721/erc721.cairo

```diff
@@ -531,14 +532,15 @@ mod ERC721Component {
     fn _check_on_erc721_received(
         from: ContractAddress, to: ContractAddress, token_id: u256, data: Span<felt252>
     ) -> bool {
-        if (DualCaseSRC5 { contract_address: to }
+        if (ISRC5Dispatcher { contract_address: to }
             .supports_interface(interface::IERC721_RECEIVER_ID)) {
             DualCaseERC721Receiver { contract_address: to }
                 .on_erc721_received(
                     get_caller_address(), from, token_id, data
                 ) == interface::IERC721_RECEIVER_ID
         } else {
-            DualCaseSRC5 { contract_address: to }.supports_interface(account::interface::ISRC6_ID)
+            ISRC5Dispatcher { contract_address: to }
+                .supports_interface(account::interface::ISRC6_ID)
```

Comment:
I think this is a good time to update this into:

```
let src5_dispatcher = ISRC5Dispatcher { contract_address: to };
if(src5_dispatcher.supports_interface(interface::IERC721_RECEIVER_ID)) { ...
```

Reply:
do you prefer :

```
if(src5_dispatcher.supports_interface(interface::IERC721_RECEIVER_ID)) { ...
```

or

```
if src5_dispatcher.supports_interface(interface::IERC721_RECEIVER_ID) { ...
```
?

Reply:
Second option